.github/workflows/codeql_analysis.yml

```diff
@@ -0,0 +1,24 @@
+name: "CodeQL Analysis"
+on:
+  push:
+    branches: [ main ]
```

Comment:
Why is this one on push when the others were cron'd?

Reply:
This is what is being done in the [Collector repo](https://github.com/open-telemetry/opentelemetry-collector/pull/2915)